bitcoin/src/blockdata/constants.rs

```diff
@@ -152,10 +195,21 @@ impl ChainHash {
         101, 225, 90, 8, 156, 104, 214, 25, 0, 0, 0, 0, 0,
     ]);
     /// `ChainHash` for testnet3 bitcoin.
+    #[deprecated(since = "0.33.0", note = "Use TESTNET3 instead")]
```

Comment:
In 04b35a617b2843e2fee7427ab12c0cac2efca782:

The `since` should be changed to 0.32.4.